packages/components/cypress.config.ts

```diff
@@ -1,7 +1,14 @@
 const { defineConfig } = require('cypress');
+const path = require('path');
 
 /** @type {import('cypress').defineConfig} */
 module.exports = defineConfig({
+  reporter: path.resolve(__dirname, "../../node_modules/mocha-ibutsu-reporter"),
+  reporterOptions: {
+    project: "frontend-components",
+    component: "some-component",
```

Comment:
I guess since this is a monorepo this is less important... but it'd be nice to indicate this is related to `RedHatInsights` or something

Reply:
We'll need a component value that aligns with our QE test data aggregation. I'm guessing it will need to match the component values available in Ibutsu.

Reply:
@catastrophe-brandon do you know which component values are available? From the reporter doc it seemed like we could pick whatever value we want - but I see what you mean with aligning with QE test data aggregation. 

Ref: https://github.com/ibutsu/mocha-ibutsu-reporter#merging-files

Reply:
@florkbr I'll defer to @catastrophe-brandon here. No clue what values are required.